Tests/Bench/Client/src/Program.cs

```diff
@@ -239,24 +266,62 @@ private static async Task Main()
 
     var resultsAvailable = Stopwatch.GetTimestamp();
 
-    foreach (var resultId in results)
+    var countRes = 0;
+
+    results.AsParallel()
```

Comment:
Do we want spawning threads and do blocking IO, or using async/await with a semaphore?

Reply:
Not today